### DISCUSSION TITLE
Styling emails

### DISCUSSION THREAD

**Discussion by rithuacharya** [strapi/strapi#8361] [Legacy Topics]
Hey, I've created a new email template "add_project" that works well for sending text information.

But I wanted to add some styling to it.

This is the options.message field of "add_project"

```
<style>
    .wrapper {
        background-color: blue;
        font-size: 20px;
    }
</style>
<div class="wrapper">
    <p>You're added to project.</p>
    <img src="<%= URL %>sample.png" />
    <p><%= URL %> Bye</p>
</div>
```
The URL is converted into the required value ( URL : new URL(strapi.config.url)), but the stylings are not applied.

Inline styling works, but that is too hard, as the actual content to be sent has too much styling.
Also tried adding link tag replacing the style tag above (by linking to a css file), but that doesn't work either.
Added image 'sample.png' in public folder but the src attribute value is https://ci3.googleusercontent.com/proxy/9NgE4vxW ... 0-d-e1-ft#http://localhost:1337/axiom-logo2.png

Not sure if it's the correct way to do it, but any help is appreciated.

1) Have internal or external css to be added (not inline as it works, but is complex to manage)
2) To load custom fonts into it
3) to add images from public folder or any other folder (used to add logo in email)

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
